fe/fe-core/src/main/java/com/starrocks/alter/AlterMVJobExecutor.java

```diff
@@ -276,6 +280,11 @@ public Void visitModifyTablePropertiesClause(ModifyTablePropertiesClause modifyT
             curProp.put(PropertyAnalyzer.PROPERTIES_PARTITION_REFRESH_NUMBER, String.valueOf(partitionRefreshNumber));
             materializedView.getTableProperty().setPartitionRefreshNumber(partitionRefreshNumber);
             isChanged = true;
+        } else if (propClone.containsKey(PropertyAnalyzer.PROPERTIES_PARTITION_REFRESH_MODE) &&
```

Comment:
can you add more tests for `create/alter mv` with this property in `CreateMaterializedViewTest`

Reply:
Fixed